dspace-api/src/main/java/org/dspace/content/comparator/NameAscendingComparator.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content.comparator;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.dspace.content.DSpaceObject;
+
+import java.util.Comparator;
+
+public class NameAscendingComparator implements Comparator<DSpaceObject>{
+
+    @Override
+    public int compare(DSpaceObject dso1, DSpaceObject dso2) {
+        if (dso1 == dso2){
+            return 0;
+        }else if (dso1 == null){
+            return -1;
+        }else if (dso2 == null){
+            return 1;
+        }else {
+            return ObjectUtils.compare(dso1.getName(),dso2.getName());
```

Comment:
I would suggest to either use `StringUtils.compareIgnoreCases(String, String)` here or to trim the Strings and convert them to either small or capital letters. Currently sorting the community/collections ends up in listing all communities/collection A-Z starting with capital letters first and then listing the communities/collection a-z starting with small letters. I would expect one alphabetical sequence, not two.